tests/testhelpers.nim

```diff
@@ -0,0 +1,25 @@
+#                Chronos Test Suite
+#            (c) Copyright 2019-Present
+#         Status Research & Development GmbH
+#
+#              Licensed under either of
+#  Apache License, version 2.0, (LICENSE-APACHEv2)
+#              MIT license (LICENSE-MIT)
+
+func toBytes*(src: openArray[char]): seq[byte] =
```

Comment:
isn't there `bytesToString` etc already?

Reply:
yep but it is declared in `http` module.

Reply:
we have stew/byteutils too ..